bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RackawareEnsemblePlacementPolicy.java

```diff
@@ -118,17 +118,21 @@ public BookieSocketAddress replaceBookie(
     }
 
     @Override
-    public List<Integer> reorderReadSequence(ArrayList<BookieSocketAddress> ensemble,
-                                             List<Integer> writeSet,
-                                             Map<BookieSocketAddress, Long> bookieFailureHistory) {
-        return super.reorderReadSequence(ensemble, writeSet, bookieFailureHistory);
+    public int[] reorderReadSequence(
+            ArrayList<BookieSocketAddress> ensemble,
+            Map<BookieSocketAddress, Long> bookieFailureHistory,
+            int[] writeSet) {
+        return super.reorderReadSequence(ensemble, bookieFailureHistory,
+                                         writeSet);
     }
 
     @Override
-    public List<Integer> reorderReadLACSequence(ArrayList<BookieSocketAddress> ensemble,
-                                                List<Integer> writeSet,
-                                                Map<BookieSocketAddress, Long> bookieFailureHistory) {
-        return super.reorderReadLACSequence(ensemble, writeSet, bookieFailureHistory);
+    public int[] reorderReadLACSequence(
+            ArrayList<BookieSocketAddress> ensemble,
+            Map<BookieSocketAddress, Long> bookieFailureHistory,
+            int[] writeSet) {
+        return super.reorderReadLACSequence(ensemble, bookieFailureHistory,
```

Comment:
Why do we need to call super? We can just not override the method

Reply:
I left it as it always was.